Runtime/Scripts/Export/GltfWriter.cs

```diff
@@ -2175,6 +2175,14 @@ unsafe int WriteBufferViewToBuffer(byte[] bufferViewData, int? byteStride = null
             }
         }
 
+        static string EscapeJson(string s)
+            => s?.Replace("\\", "\\\\")
```

Comment:
This is problematic when it comes to memory resource consumption since every instance of `Replace` creates a copy of the original string and returns that copy. So this creates 6 copies of the original string in any case.